resources/welcome/index.html

```diff
@@ -3,18 +3,34 @@
 
 <head>
     <link rel="stylesheet" href="{{root}}/resources/welcome/main.css">
+    <script src="{{root}}/resources/welcome/highlight-9.13.1.min.js"></script>
+    <script>hljs.initHighlightingOnLoad();</script>
 </head>
 
 <body class="vscode-dark">
     <header>
         <div id="title-area">
-            <h1>Azure IoT Hub Toolkit Extension</h1>
-            <p>Interact with Azure IoT Hub, IoT Device Management, IoT Edge Management, IoT Hub Code Generation.</p>
+            <h1>Azure IoT Tools</h1>
+            <p>Azure IoT Tools is a collection of popular extensions that that make it easy to discover and interact with Azure IoT Hub that power your IoT Edge and device applications.</p>
         </div>
     </header>
     <div id="content">
         <div id="main-content">
             <div>
+                <div id="whats-new">
+                    <h2>What's new</h2>
+                    <ul>
+                        <li>
+                            <a href="https://github.com/Microsoft/vscode-azure-iot-toolkit/blob/master/CHANGELOG.md">Azure IoT Hub Toolkit Update</a>
```

Comment:
We may no want to add `Update` for each extension, it is redundant

Reply:
removed